en/middleware/drivers.md

```diff
@@ -112,3 +112,33 @@ and `devtype` is decoded according to:
 #define DRV_RNG_DEVTYPE_MB12XX   0x31
 #define DRV_RNG_DEVTYPE_LL40LS   0x32
 ```
+
+## Debugging
+
+For general debugging topics see: [Debugging/Logging](../debug/README.md).
+
+### Verbose Logging
+
+Drivers (and other modules) output minimally verbose logs strings by default (e.g. for `PX4_DEBUG`, `PX4_WARN`, `PX4_ERR`, etc.).
+
+Log verbosity is defined at build time using the `RELEASE_BUILD` (default), `DEBUG_BUILD` (verbose) or `TRACE_BUILD` (extremely verbose) macros.
+
+Change the logging level using `COMPILE_FLAGS` in the driver `px4_add_module` function (**CMakeLists.txt**).
```

Comment:
fyi another (more local) variation is to add `#define DEBUG_BUILD` at the very top of a .cpp file (before includes).

Reply:
@bkueng David pointed out I should not add content without testing it, which I do not plan on doing. Lorenz suggested that I should provide more "support" and less "do" as a better use of my time. 

Upshot, please feel encouraged to add this sort of note yourself and then get me to comment. Definitely more efficient that me doing this: https://github.com/PX4/Devguide/pull/983